web_src/less/_base.less

```diff
@@ -238,6 +238,14 @@ a.muted:hover,
   border-color: var(--color-primary) !important;
 }
 
+.ui.action.input:not([class*="left action"]) > .ui.dropdown.selection {
```

Comment:
```suggestion
/* dropdown on repo search and explore code */
.ui.action.input:not([class*="left action"]) > .ui.dropdown.selection {
```

Should really comment these.

Reply:
Yes, but if we notice this causing issues further down the road, we at least know what it's supposed to be targeting. Of course if you can fabricate a more specific selector, that'd also be an option.